### PR TITLE
Add Swagger service support

### PR DESCRIPTION
**Pull request by LinuxSuRen**
### What type of PR is this?

/kind chore

### What this PR does / why we need it:
With help of the Swagger UI, people can understand the APIs.

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:

```
```

You can see the following screenshot:

image

### Does this PR introduce a user-facing change??

```release-note
Add Swagger service support
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -113,6 +113,9 @@ docker-build-push-tools: docker-build-tools docker-push-tools
 
 docker-build-push: docker-build-push-apiserver docker-build-push-controller
 
+swagger-ui:
+	git clone git@github.com:swagger-api/swagger-ui.git -b v2.2.10 bin/swagger-ui
```
Comment (JohnNiang) [inline Makefile:117]: How about setting option `depth` to `1`?

File: config/dockerfiles/apiserver/Dockerfile [hunk 2]
```diff
@@ -17,11 +17,15 @@ COPY pkg pkg/
 # Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o apiserver cmd/apiserver/apiserver.go
 
+# download Swagger UI files
+RUN git clone https://github.com/swagger-api/swagger-ui -b v2.2.10
```
Comment (JohnNiang) [inline config/dockerfiles/apiserver/Dockerfile:21]: How about setting option `depth` to `1` and deleting `.git` folder to make the container thinner?

**Review by JohnNiang — COMMENTED** [review]
Swagger UI is a great tool for API documentation. Thanks for your time. I've left two comments below, please take a look at it.

File: config/dockerfiles/apiserver/Dockerfile (hunk 2 above)
Comment (LinuxSuRen) [reply config/dockerfiles/apiserver/Dockerfile:21]: No need to delete any directories. This is a multi-stage Dockerfile. I only copy the `dist` directory.

File: Makefile (hunk 1 above)
Comment (LinuxSuRen) [reply Makefile:117]: Good idea.

**Review by JohnNiang — APPROVED** [review]
/lgtm
/label tide/merge-method-squash

**Pull request merged**